docs/blog/2019-06-21-profitable-agency-wordpress/index.md

```diff
@@ -0,0 +1,117 @@
+
+The cheaper hosting for static sites, like the ones Gatsby can produce, can be a game changer for your business model. That's because the client is willing to pay for an all-inclusive package, and doesn't care if you use an inexpensive static site host, or a costly WordPress host. By making the right choice your profit can increase from day one, and accelerate if you keep the project over months or years.
+
+I'm old enough to remember using Movable Type and sneering at WordPress. I worked at Vibe Magazine and didn't choose WordPress as our CMS because at the time it couldn't handle our requirement for multiple authors. However, Movable Type eventually ceded their #1 position to WordPress, in large part due to hosting costs. Hosting for a PHP CMS like WordPress was cheaper and simpler than finding a home for the Perl-powered Movable Type.
+
+Static sites in general, and Gatsby in particular will win because the cost of getting started, and keeping going is so low. And as a bonus, static site hosting truly is more secure and simpler to use. It's rare that a new way of doing things ticks the boxes of FAST, GOOD, and CHEAP with no compromise. But that perfectly describes [the hosting options for Gatsby](https://www.gatsbyjs.org/docs/deploying-and-hosting/).
+
+## Break-Fix happens less, and is solved faster
+
+A core service you probably offer your clients is break-fix. That is, when something breaks on their site, you swoop in and fix it. It's the biggest selling point for a client. They are willing to pay your monthly retainer for the insurance of having a professional on hand if anything goes wrong.
+
+And it's the biggest driver of your profit. You're not ashamed to admit that breakage doesn't happen that often. That's because your initial WordPress setup and selection of themes and plugins is considerate. So, your clients pay monthly for a situation that only arises maybe twice a year.
+
+But what keeps you up at night? What makes you sweat when you think about it?
+
+I know. It's that random bug or issue that takes hours or days longer to fix than you ever planned for. It could be a problem with WordPress core, or the server, the hosting provider, or a plugin, or the way two plugins are interacting with each other.
+
+I once discovered a bug caused by a page builder plugin update that prevented a theme from displaying comments. I spent half a day sussing out the culprit, then another day getting the page builder's support team to take responsibility and recognize it as a true bug. Then a week later it was fixed.
+
+Because WordPress plugins are writing to the same database, using the same platform functions, and jockeying for space in the same admin area, the permutations of potential problems is high.  And solving issues can take a lot of time.  Also, unless you lock down your WordPress projects with as little user privileges as possible, then you're sure to get a call every so often about something your client broke himself.
```

Comment:
```suggestion
Because WordPress plugins are writing to the same database, using the same platform functions, and jockeying for space in the same admin area, the permutations of potential problems is high.  And solving issues can take a lot of time.  Also, unless you lock down your WordPress projects with as little user privileges as possible, then you're sure to get a call every so often about something your client broke themself.
```